posthog/api/instance_status.py

```diff
@@ -44,6 +45,10 @@ def list(self, request: Request) -> Response:
 
         metrics.append({"key": "posthog_git_sha", "metric": "PostHog Git SHA", "value": GIT_SHA})
 
+        metrics.append(
+            {"key": "chart_version", "metric": "Chart version", "value": os.getenv("CHART_VERSION", "Undefined"),}
```

Comment:
Note: This is slightly misleading. People can use helm charts without using clickhouse helm charts. :)